Resources/js/translator.js

```diff
@@ -53,6 +53,17 @@ var Translator = (function(document, undefined) {
         var _locale = locale || currentLocale || localeFallback,
             _domain = domain;
 
+        /**
+         * If the _locale looks like nl_NL or nl-NL
+         * Try the language as the locale
+         */
+        if (undefined == _messages[_locale]) {
+            var tryLanguage = _locale.match(/^([a-z]{2})(_|-)[A-Z]{2}$/)
```

Comment:
and btw, the ``localeFallback`` could be ``en_US`` too, which should itself fallback to ``en`` to mimic the Symfony behavior